aws/rust-runtime/aws-inlineable/src/s3_express.rs

```diff
@@ -702,6 +702,11 @@ pub(crate) mod runtime_plugin {
                             value,
                         )));
                     }
+                    Ok(value) => {
```

Comment:
The warning is going to prevent fallback to the profile file value once that's implemented. You can fix that later though.

Reply:
Yeah, whether specifying an unexpected value for the env variable is penalized or not (i.e. not falling back on the profile file value because you just specified the env variable) needs some discussion.